docs/learn/fundamentals/fees-resource-limits-metering.mdx

```diff
@@ -177,8 +175,6 @@ If the contract execution concludes within the specified resource limits, the me
 
 For the past three hours, inclusion fee statistics on the Mainnet network can be seen below.
```

Comment:
We probably want to take this out too? Or, maybe just `{/* put it into a comment? */}`?